images/macos/macos-13-Readme.md

```
@@ -1,98 +1,105 @@
+| Announcements |
+|-|
+| [[ macOS ] .NET 7.x will be removed from macOS-13 and macOS-14 images on February 24,2025.](https://github.com/actions/runner-images/issues/11501) |
+| [macOS 15 (Sequoia) will be generally available in GitHub Actions and Azure DevOps from March, 18](https://github.com/actions/runner-images/issues/11486) |
+| [[macOS] Ruby version 3.0 will be removed from the images on Feb 17, 2025 and 3.3 will be set as default. ](https://github.com/actions/runner-images/issues/11345) |
```

markdownlint-cli2 (0.17.2)

Spaces inside link text
null

(MD039, no-space-in-links)